service/routes.py

```diff
@@ -22,3 +22,38 @@ def index():
         ),
         status.HTTP_200_OK,
     )
+
+
+@app.route("/wishlists/<int:wishlist_id>", methods=["GET"])
+def get_wishlists(wishlist_id):
+    # TODO: implement this route
+    # placeholder here so create_wishlists can return a location
+    pass
+
+
+@app.route("/wishlists", methods=["POST"])
+def create_wishlists():
+    app.logger.info("Request to create a wishlist")
+    check_content_type("application/json")
+    wishlist = Wishlist()
+    wishlist.deserialize(request.get_json())
+    wishlist.create()
+    message = wishlist.serialize()
+    location_url = url_for("get_wishlists", wishlist_id=wishlist.id, _external=True)
```

Comment:
Shouldn't it be 
```suggestion
    location_url = url_for("**create_wishlists**", wishlist_id=wishlist.id, _external=True)
```
instead?

Changing value to 'create_wishlists' works when I test with Advanced Rest Client. Otherwise it doesn't.